src/main/java/edu/rice/habanero/benchmarks/BenchmarkRunner.java

```diff
@@ -45,13 +41,20 @@ public static void runBenchmark(final String[] args, final Benchmark benchmark)
             System.exit(1);
         }
 
-        final String benchmarkName = benchmark.name();
-        if (benchmarkName.contains("Akka")) {
+        if (benchmark.name().contains("Akka")) {
             AkkaActorState.initialize();
         }
 
+        if (no_evaluation) {
+            for (int i = 0; i < iterations; i++) {
```

Comment:
Should we force `iterations` to be equal to 1 when `no_evaluation` mode is enabled?